delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ProjectsRoutes.scala

```diff
@@ -137,11 +137,11 @@ final class ProjectsRoutes(
                     // Deprecate/delete project
                     (delete & pathEndOrSingleSlash) {
                       parameters("rev".as[Int], "prune".?(false)) {
-                        case (rev, true) if config.deletion.enabled =>
+                        case (rev, true)  =>
```

Comment:
I removed it from there as the deletion can be triggered from the deletion plugin and other places, so I included it in the state machine as it is way safer

Reply:
The behaviour changes yes, as you will now get an error if project deletion is disabled.
But it is cleaner that way in my opinion as delete and deprecate are not the same thing